docs/absRelTime.md

```diff
@@ -0,0 +1,33 @@
+## jquery.flot.absRelTime.js
```

Comment:
Maybe a small table like this one: https://github.com/ni-kismet/webcharts-drive-test/blob/master/node_modules/ni-webcharts/docs/API.md#ni-cartesian-axis